android/engine/src/test/java/org/smartregister/fhircore/engine/p2p/dao/BaseP2PTransferDaoTest.kt

```diff
@@ -253,6 +253,33 @@ class BaseP2PTransferDaoTest : RobolectricTest() {
     assertEquals("Patient", search.type.name)
   }
 
+  @Test
+  fun `getDefaultDataTypes() returns correct list of datatypes`() {
```

Comment:
Can be renamed as per the recommendation in the [Code-Testing- wiki](https://github.com/opensrp/fhircore/wiki/Code-Testing)